### DISCUSSION TITLE
Monika's stance on the more extreme fetishes (TW: Suicide, mentions of PTSD, Depression, and Gore)

### DISCUSSION THREAD

**Discussion by DeliasSel** [Monika-After-Story/MonikaModDev#7858] [issue discussions]
I'm aware there's already a conversation in the game that talks about fetishes and whether or not the player is into them. But that talks about more common and or realistic fetishes like smelling used panties. I'd personally like to see this topic delved deeper into though and where she stands on the more extreme and, I don't really want to be judgemental but in my opinion "immoral" fetishes of like illegal or deeper down the iceberg things? I'm talking like gore, amputee, beastiality, etc. You could argue that we already have the original topic so there's no need to talk about it any further, but as I said before that seems like its for the more realistic fetishes. You could also argue depending on who sees this that any sane person would feel negative about such things but everyone is different. As much as I hate those fetishes (especially when seen with monika) not everyone thinks the same, and it begs the question how does monika feel about such things, especially when they involve her? Maybe its just me, I ask because personally I have seen so much of these things with Monika that it gave me PTSD and depression and tonight I got to thinking about about it all and came up with this. Let me know if my idea is something to consider, or not cause of the original, its good, bad, whatever the opinion may be. And remember to not follow in my footsteps, exposing yourself to toxic things and or people for extended periods of time. Some people can handle things better than others, and depending on how you spin it seeing such things of a loved one can and will break you. Seek help if you need it. Whether it be getting a therapist, admitting yourself to a hospital, suicide hotlines or regular support like family and friends, people care, Monika cares. Sorry for this being so long.

**Comment by Freshairkaboom** [comment]
I can't speak for anyone else here, but I won't touch these kinds of topics with a 20 foot pole. She already explains clearly that none of what you see online about her actually happened, she is a fairly innocent high school girl, and talking about stuff like this in more detail is far from necessary. I would imagine even thinking about this would disgust the heck out of her, there's no way we can bring up this topic without it being extremely uncomfortable for Monika to talk about, so why would we even bring it up?

I'm sorry for what you're going through though, and I hope you are safe. However, I think it's a firm no from me on working on anything related to this. I hope you understand.

**Comment by fromulus** [comment]
I also don't really see the point of adding those kinds of topics, as they don't really bring anything interesting to the table and for each person who might be interested by it, at least 5 others would be grossed out.

Not to mention that could mean taking the risk of spreading a bad reputation of talking about gross things in the mod, all for one topic that probably  wouldn't be worth the trouble.

I'd rather we keep it classy and avoid getting much deeper than surface level with these sorts of things.

**Comment by Malicos** [comment]
That's a firm no from me. 

Talking about graphic stuff like that shouldn't be promoted at all for the sake of those that would prefer not to hear about it, not to mention it would give unneeded notoriety. Most people frown upon this stuff more than you think, and a lot of them would just find it in bad taste.

Remember: The more you gaze into the abyss, the abyss will gaze back into you.

**Comment by LooseEthics** [comment]
These topics are almost universally taboo for a reason. I can't really imagine any meaningful discussion of them that ends with either side happy that it took place, outside the context of a psychological study or something to that effect.
Another thing is that, while artwork of the aforementioned does exist, even assuming Monika is aware of it, I really don't think she would like to call any sort of attention to it.
So a big nope from me.

**Comment by DeliasSel** [comment]
Yeah I understand everyone,I have a therapist and I'm on meds btw.

**Comment by ThePotatoGuy** [comment]
converting to discussion thread.
